### DISCUSSION TITLE
How pass access_token from directus to another non-directus app?

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#6256] [Archive]
Hi, i'm trying to make my own webapp. 

User have to press the button inside the directus form and then it passes him/her to my app.
My app and the directus are on the **same domain**.

How to save the access_token for my app?

**Comment by rijkvanzanten** [comment]
I'm not sure if I fully understand what you're talking about, but you could make a new field on `directus_users` that stores a custom token for your app 👍🏻

**Comment by IlyaPetrovM** [comment]
I've solved the problem. 
I created directus module and used all access_token magic "out of the box".
